### PR TITLE
Additional functions to Host and Compute Resource

### PR DESCRIPTION
**Pull request by jyejare**
$subject

### CODE REVIEW

**Comment by jyejare** [comment]
@mirekdlugosz , Yeah I need to update the PR for it. It was a raw PR btw. You were too quick on it. :)

**Comment by jyejare** [comment]
@mirekdlugosz , The fact about the disassociate is, first I duplicated it and then I removed the previous one. There is no change in it at all.

**Comment by jyejare** [comment]
@mirekdlugosz Feel free to revisit. I have added the tests, plus one more def of 'available_networks()' has been added.

File: nailgun/entities.py [hunk 1]
```diff
@@ -899,6 +899,35 @@ def __init__(self, server_config=None, **kwargs):
...
+            /api/compute_resources/:id/available_images
+        available_flavors
+            /api/compute_resources/:id/available_flavors
+        available_zones
+            /api/compute_resources/:id/available_zones
+        available_networks
+            /api/compute_resources/:id/available_networks
+        images
+            /api/compute_resources/:id/images
+
```
Comment (vijay8451) [inline nailgun/entities.py]: Please add support for below as well :
``
 /api/compute_resources/:id/associate
``

**Comment by vijay8451** [comment]
> @mirekdlugosz , The fact about the disassociate is, first I duplicated it and then I removed the previous one. There is no change in it at all.

@jyejare looks you missed the position of that ... earlier it was after `def search` but now it is  after `def  module_streams`

**Comment by jyejare** [comment]
@mirekdlugosz @vijay8451 Done. Please merge this on priority.

**Comment by mirekdlugosz** [comment]
Code looks OK on surface, but we can't merge it with travis failing.

**Comment by jyejare** [comment]
@mirekdlugosz , Didnt Looked at Travis, Fixed now.

**Pull request merged**